boards/posix/native_posix/doc/board.rst

```diff
@@ -35,6 +35,8 @@ It has only been tested on Linux, but should also be compatible with macOS.
    This port will **not** work in Windows Subsystem for Linux (WSL) because WSL
    does not support native 32-bit binaries.
 
+.. _native_important_limitations:
```

Comment:
@dbkinder : Are these 2 extra references needed after your changes? Or should I just delete them?